pkg/util/merge/merge_test.go

```diff
@@ -101,25 +101,11 @@ func TestMergeServices(t *testing.T) {
 				Selector: map[string]string{"test1": "true", "test2": "true"},
```

Comment:
how do we handle selectors with the same key but different values? Should we add a test for this?